src/ops/conv2d_test.ts

```diff
@@ -74,6 +74,33 @@ describeWithFlags('conv2d', ALL_ENVS, () => {
     expectArraysClose(result, [2, 4, 6, 8]);
   });
 
+  it('packed x=[2,1,2] f=[1,1,2,2] s=1 d=1 p=0', () => {
```

Comment:
Could you move this test into a `describeWithFlags` block that applies only to `WEBGL_ENVS`? I think this is currently under `ALL_ENVS`